Client/Frontend/Browser/BrowserViewController/BrowserViewController+ToolbarDelegate.swift

```diff
@@ -620,19 +620,38 @@ extension BrowserViewController: ToolbarDelegate {
             addTabAlertActions().forEach(controller.addAction)
         }
         
+        if tabManager.getCountForWebsite() > 0 {
+            controller.addAction(
+                UIAlertAction(title: Strings.bookmarkAllTabsTitle,
+                              style: .default, handler: { [unowned self] _ in
+                
+                let addBookMarkController = AddEditBookmarkTableViewController(
+                    mode: BookmarkEditMode.addFolderUsingTabs(title: Strings.savedTabsFolderTitle, tabList: tabManager.allTabs))
```

Comment:
What happens if you have 2 tabs opened in regular mode, 3 tabs in private mode and you add them?

Please double check this code works well in this case, priv mode tabs and regular tabs should be added and counted separately

Reply:
perhaps `tabsForCurrentMode` should be the correct getter, not `tabManager.allTabs`, haven't checked it though

Reply:
`tabsForCurrentMode` is the correct getter just checked.

Fix at https://github.com/brave/brave-ios/pull/4133/commits/cbe4bfce05e9b1424d52e36938def7cc475ff535